app/controllers/api/v1/about_controller.rb

```diff
@@ -0,0 +1,9 @@
+module Api
+  module V1
+    class AboutController < ApiController
+      def index
```

Comment:
Maybe? I think the user does not needs to be authorized to do this action, wdyt?

```suggestion
      skip_before_action :authenticate_user!

      def index
```